config.json

```diff
@@ -1308,7 +1308,17 @@
         "math"
       ]
     },
-
+    {
+      "slug": "resistor-colors",
+      "uuid": "de800041-3dcc-41b9-b101-7314ff685c93",
+      "core": false,
+      "unlocked_by": "pangram",
```

Comment:
Should be unlocked_by: "resistor-color" (#640) as per the track-tool.

```suggestion
      "unlocked_by": "resistor-color",
```

Reply:
We don't have `resistor-color`. Might need to implement that first.

Reply:
Ohh ok thats what #640 is for.

Reply:
Also need to make this core as per #641

Reply:
You got it!

Reply:
#640 is in #648

Reply:
@tejasbubane want to move this up in `config.json` and make it `core`? If you don't have time I can take care of it.